commands/utils/utils_test.go

```diff
@@ -195,3 +195,68 @@ func verifyEnv(t *testing.T) (server config.ServerDetails, restoreFunc func()) {
 	}
 	return
 }
+
+func TestIsValidBranchName(t *testing.T) {
```

Comment:
TestIsValidBranchName --> TestValidatedBranchName
(the test name should match the function name, which is `validateBranchName`).